server/models/handlers/account.js

```diff
@@ -1,9 +1,24 @@
 /* The goal: Take in the user input from the Account Creation form in 
 the front end UI in order to create a user account in the backend*/
 
-async function createUserAccount(firstName, lastName, type, studentID, email, password, phone) {
-    
-    const user = {
+async function createUserAccount(first_name, last_name, type, student_id, email, password, phone) {
+     // Searches the student table to see if a student account exists for the given data.
+
+     If userType = userType.student {
+     const user = await studentModel.insertStudentAccount(first_name, last_name, type, student_id, email, password, phone);
```

Comment:
Hi Nancy,
In addition, I would recommend generating an `accessToken` value in this function for insertion into the database.

Reply:
Done :)

Reply:
Hi Nancy,
Not sure if it was not pushed successfully but I do not see this change in the file. May you please confirm if those changes were pushed?

Reply:
Hi Amy, you can see the changes in line 10 of `server/models/handlers/account.js.` Please confirm if this takes care of your suggestion!

Reply:
Hi Nancy,
Yes, that works. However, I would am noticing some syntax errors (has a red underline) in your code (a screenshot is shown below). Please take a look and address them please. 
![image](https://user-images.githubusercontent.com/32720322/98050683-3ac8bf00-1e00-11eb-8a22-e3c0ea6675fc.png)